app/src/main/java/chat/rocket/android/authentication/loginoptions/presentation/LoginOptionsPresenter.kt

```diff
@@ -170,13 +170,13 @@ class LoginOptionsPresenter @Inject constructor(
         }
     }
 
-    private fun setupConnectionInfo(serverUrl: String) {
+    fun setupConnectionInfo(serverUrl: String) {
```

Comment:
Is there any other way to unit test these private methods, other than make them public?  Will this be ok with Fillipe?  Can you ask him about that maybe?

Reply:
Sure I will ask about this